### PR TITLE
Write a R program to check anagram string #6657

### LINKED ISSUE #6657: Write a R program to check anagram string
### Description

Write a R program to check anagram string

Two strings are said to be anagrams if they make a meaningful word by rearranging or shuffling the letters of the string. In other words, we can say that two strings are anagrams if they contain the same characters but in different order.

```txt
Input  : pan, nap
Output : Anagram Strings
```

<sub>Contributed by - @sahdev77</sub>
<details>
<summary>How to contribute</summary>

- Save the solution in `write_a_program_to_check_anagram_string.r` file
- Add `write_a_program_to_check_anagram_string.r` file in `program/write-a-program-to-check-anagram-string` folder

</details>

### PR DESCRIPTION
**Pull request by Preetiraj3697**
Closes https://github.com/codinasion/codinasion/issues/4343 (Write a R program to check anagram string )
Closes https://github.com/codinasion/codinasion/issues/4120 (Write a F# program to print elements of an array)
Closes https://github.com/codinasion/codinasion/issues/4078 program to merge two strings

### CODE REVIEW

**Review by hi-Kartik2004 — CHANGES_REQUESTED** [review]
Hi @Preetiraj3697, firstly thanks for making a pull request which closes 2 issues, but your comment says you are fixing a single issue, can you link all the issues you're fixing to the first comment?

Format: Closes  (issue number goes here)

File: program/print-elements-of-an-array/PrintElementsOfAnArray.fs
Comment (Preetiraj3697) [inline program/print-elements-of-an-array/PrintElementsOfAnArray.fs:1]: Close issue #5470

File: program/check-anagram-string/write_a_program_to_check_anagram_string.r
Comment (Preetiraj3697) [inline program/check-anagram-string/write_a_program_to_check_anagram_string.r:1]: Close issue #6657

**Comment by Preetiraj3697** [comment]
> Hi @Preetiraj3697, firstly thanks for making a pull request which closes 2 issues, but your comment says you are fixing a single issue, can you link all the issues you're fixing to the first comment?
> 
> Format: Closes (issue number goes here)

I add it. 
Close issue codinasion/codinasion#4343  (Write a R program to check anagram string )
Close issue codinasion/codinasion#4120 (Write a F# program to print elements of an array)

File: program/merge-two-strings/MergeTwoStrings.fs
Comment (Preetiraj3697) [inline program/merge-two-strings/MergeTwoStrings.fs:1]: Close issue #5428 program to merge two strings

**Review by hi-Kartik2004 — APPROVED** [review]

**Pull request merged**